cmd/oras/root/discover.go

```diff
@@ -188,7 +188,9 @@ func printDiscoveredReferrersTable(refs []ocispec.Descriptor, verbose bool) {
 	for _, ref := range refs {
 		print(ref.ArtifactType, ref.Digest)
 		if verbose {
-			printJSON(ref)
+			if err := printJSON(ref); err != nil {
+				fmt.Printf("Error printing JSON %v", err)
+			}
```

Comment:
should add an error into return value of `printDiscoveredReferrersTable` and return in the caller function